app/src/components/market/sections/market_buy/market_buy.tsx

```diff
@@ -194,6 +200,13 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
           probabilities={probabilities}
           showPriceChange={amount.gt(0)}
         />
+        <WarningMessageStyled
+          description={
+            "Before trading on a market, make sure that its outcome will be known by its resolution date and it isn't an"
+          }
+          hyperlink={'./rules.pdf'}
```

Comment:
```
          hyperlink={'./rules.pdf'}
```

Maybe you can put this and `faq.pdf` in `app/src/common/constants.ts`. Will be handy if / when you have to change it.

Reply:
done!